src/site/antora/modules/ROOT/pages/manual/lookups.adoc

```diff
@@ -966,7 +966,10 @@ include::partial$features/servlet-support.adoc[]
 [#third-party]
 == Third-party lookups
 
-The following additional lookups are available from third-party vendors:
+[IMPORTANT]
+====
+These lookups are provided by **third-party** vendors and are not maintained by the Apache Software Foundation.
```

Comment:
```suggestion
These lookups are provided by **third-party** vendors and are not maintained by the link:{logging-services-url}[Apache Logging Services] (Log4j, Log4cxx, Log4net) project.
```

Reply:
Log4j releases are an act of the Apache Software Foundation, so I think it is important to visually differentiate software for which the ASF is **not** responsible. If we want to put the limit closer to us, I am fine with it too.

Reply:
ASF doesn't **maintain** the code, ALS does. ASF gets only involved in the context of governance. Saying ASF maintains Log4j, etc. not only gives incorrect information, but leads users to donate to ASF while they actually want to support Log4j maintainers. During Log4Shell a substantial amount of donations are made to ASF and ALS, _the maintainers_, got 0 (zero). Hence, could you please make it explicit that Log4j maintainers (i.e., ALS) maintain Log4j, not ASF?

Reply:
I already did.